### ISSUE TITLE
RUSH 2049 doesnt play audio

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#287]
Originally reported on Google Code with ID 288

```
Describe your system:
 - Linux distribution: Debian sid
 - Machine type: amd64
 - Mupen64Plus version: svn1416
 - Plugins used: glide64, hacktarux rsp

For visual artifacts, give:
 - MD5: 02B16AC23998F78F09AF6513F4ACB664
 - No sound in game

There seems to be a problem inside the ucode emulation of the rsp. It
works as expected in z64's rsp emulation. The audio ucode is currently
ucode3.
```

Reported by `sven@narfation.org` on 2009-10-19 14:53:58

**Comment by richard42** [comment]
Reported by `richard42g` on 2010-01-13 13:22:26
- **Labels added**: Type-Defect, Component-RSP-HLE

**Comment by richard42** [comment]
```
It should work fine with --rsp mupen64plus-rsp-z64-hlevideo
```

Reported by `sven@narfation.org` on 2011-09-02 21:37:23

**Comment by richard42** [comment]
Reported by `sven@narfation.org` on 2013-12-18 12:11:07
- **Status changed**: `Duplicate`
- **Merged into**: #99
